medical_patient_dob/views/medical_patient_dob_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+ <record id="medical_patient_dob_view" model="ir.ui.view">
+        <field name="name">medical.patient.dob.view</field>
+        <field name="model">medical.patient</field>
+        <field name="inherit_id" ref="medical.medical_patient_view_search" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='birthdate_date']" position="replace">
+                <field name="birthdate" />
```

Comment:
We should instead just make the change in whatever file actually introduces the field into the search, which is likely medical core